system/include/emscripten/fetch.h

```diff
@@ -53,6 +53,9 @@ extern "C" {
 // to test or wair for its completion.
 #define EMSCRIPTEN_FETCH_WAITABLE 128
 
+// If specified, the response headers will be loaded into the fetch struct.
+#define EMSCRIPTEN_FETCH_RESPONSE_HEADERS 256
```

Comment:
Let's not add a #define for this, it should not be needed.

Reply:
Removed.